playbooks/network.yaml

```diff
@@ -125,11 +125,11 @@
       ansible.builtin.set_fact:
         checkpoint: "{{ (nmstateset.stdout_lines | last).split()[1] }}"
 
-    - name: Set fact for nmstate checkpoing on RHEL9
+    - name: Set fact for nmstate checkpoint on RHEL9
       when:
         - ansible_facts.distribution_major_version == "9"
       ansible.builtin.set_fact:
-        checkpoint: "{{ (nmstateset.stderr_lines | last).split()[-1] }}"
+        checkpoint: "{{ (nmstateset.stdout_lines | select('search', 'Checkpoint: NetworkManager\\|') | last).split()[1] }}"
```

Comment:
I've tested RHEL 9.2 and OSP 17.1 and this wasn't needed.

Reply:
regarding the change to Set fact for nmstate checkpoint on RHEL9, this was required when i did a clean install of OSP 17.0 on RHEL9.0

it was not required on centos stream

it seems particular to 17.0 on RHEL9

Reply:
ack, will have to test it on RHEL 8.4 and RHEL 9.2 too to make sure it's compatible.

Reply:
I have added handlers for rhel 9.0 + osp 17.0 and rhel 9.2 (and onwards) and osp 17.1

Reply:
If it's only a RHEL 9.0 issue, I suggest that we don't add this code, to reduce complexity.

I think dev-install is going to support 16.2 and OSP 17.1 in the longterm, we can't add support for every version of OSP, especially the unsupported ones (17.0).